src/Applications/GEOSctm_App/CMakeLists.txt

```diff
@@ -38,12 +38,21 @@ install (
    DESTINATION etc
    )
    
+# Did we build with Hydrostatic?
```

Comment:
Does this comment really help?   If it does, then maybe "HYDROSTATIC" should be recast as "BUILD_WITH_HYDROSTATIC"?  Else, delete the comment.

Reply:
I could change this to `BUILD_WITH_HYDROSTATIC` but I'd need to change I think five other repositories at the same time. But I'll delete the comment.